### PR TITLE
{toolchain} GCC 7.3.0 + binutils 2.30

### PR DESCRIPTION
**Pull request by jhein32**

### CODE REVIEW

File: easybuild/easyconfigs/g/GCCcore/GCCcore-7.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    'mpfr-%s.tar.bz2' % mpfr_version,
+    'mpc-1.0.3.tar.gz',
+    'isl-0.19.tar.bz2',
+]
+patches = [
+    ('mpfr-%s-allpatches-20180412.patch' % mpfr_version, '../mpfr-%s' % mpfr_version),
+    'GCCcore-6.2.0-fix-find-isl.patch',
+]
+checksums = [
+    'fa06e455ca198ddc11ea4ddf2a394cf7cfb66aa7e0ab98cc1184189f1d405870',  # gcc-7.2.0.tar.gz
```
Comment (boegel) [inline easybuild/easyconfigs/g/GCCcore/GCCcore-7.3.0.eb]: @jhein32 Comment still says `7.2.0`, please use `eb --inject-checksums` to correct this (don't do it manually)

**Comment by jhein32** [comment]
On 14 Apr 2018, at 10:38, Kenneth Hoste <notifications@github.com<mailto:notifications@github.com>> wrote:


@boegel requested changes on this pull request.

________________________________

In easybuild/easyconfigs/g/GCCcore/GCCcore-7.3.0.eb<https://github.com/easybuilders/easybuild-easyconfigs/pull/6153#discussion_r181546107>:

> +    'http://gcc.cybermirror.org/infrastructure/',  # HTTP mirror for GCC dependencies
+    'http://isl.gforge.inria.fr/',  # original HTTP source for ISL
+]
+sources = [
+    'gcc-%(version)s.tar.gz',
+    'gmp-6.1.2.tar.bz2',
+    'mpfr-%s.tar.bz2' % mpfr_version,
+    'mpc-1.0.3.tar.gz',
+    'isl-0.19.tar.bz2',
+]
+patches = [
+    ('mpfr-%s-allpatches-20180412.patch' % mpfr_version, '../mpfr-%s' % mpfr_version),
+    'GCCcore-6.2.0-fix-find-isl.patch',
+]
+checksums = [
+    'fa06e455ca198ddc11ea4ddf2a394cf7cfb66aa7e0ab98cc1184189f1d405870',  # gcc-7.2.0.tar.gz


@jhein32<https://github.com/jhein32> Comment still says 7.2.0, please use eb --inject-checksums to correct this (don't do it manually)


Why should I not do this manually?  Will it create issues?

J.

**Comment by boegel** [comment]
@jhein32 Well, no, you may make mistakes, like not updating the comment. `--inject-checksums` will always get it right.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node2103.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/03acaec76784417640d5c7e7bd37ec28 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (9 easyconfigs in this PR)
node2583.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/bca811ecd02b850748d5b25f74d45ddc for a full test report.

**Comment by jhein32** [comment]
you got email regarding the log

**Comment by boegel** [comment]
Relevant error from the log:

```
../../../mpc/src/mul.c:175:1: error: conflicting types for ‘mpfr_fmma’
 mpfr_fmma (mpfr_ptr z, mpfr_srcptr a, mpfr_srcptr b, mpfr_srcptr c,
 ^
In file included from ../../../mpc/src/mpc.h:25:0,
                 from ../../../mpc/src/mpc-impl.h:30,
                 from ../../../mpc/src/mul.c:22:
/local/easybuild/build/GCCcore/7.3.0/dummy-/gcc-7.3.0/mpfr/src/mpfr.h:731:21: note: previous declaration of ‘mpfr_fmma’ was here
 __MPFR_DECLSPEC int mpfr_fmma (mpfr_ptr, mpfr_srcptr, mpfr_srcptr, mpfr_srcptr,
                     ^
make[4]: *** [mul.lo] Error 1
```

This points to https://www.mail-archive.com/gcc@gcc.gnu.org/msg84013.html, and the follow-up which suggests that this was a bug in MPC.

@jhein32 Can you try again with MPFR 4.0.1 and MPC 1.1.0 (and also update all other libraries to latest)?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node2534.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/61cc7c0b9a6a54659863226d3331db22 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @jhein32!

**Pull request merged**